### ISSUE TITLE
Question about the loss of 'Fog Style Matching Loss'

### ISSUE THREAD

**Issue by tiamjiakun** [sohyun-l/fifo#6]
The Fog Style Matching Loss in the paper：
\mathcal{L}_{\mathrm{fsm}}^{l}\left(\mathbf{f}^{a, l}, \mathbf{f}^{b, l}\right)=\frac{1}{4 d_{l}^{2} n_{l}^{2}} \sum_{i=1}^{d_{l}}\left(\mathbf{f}_{i}^{a, l}-\mathbf{f}_{i}^{b, l}\right)^{2}

The code of Fog Style Matching Loss:
In main.py:
 ...      ...
393        fog_factor_b = fogpassfilter(vector_b_gram)
394        fog_factor_a = fogpassfilter(vector_a_gram)
395        half = int(fog_factor_b.shape[0]/2)
397        layer_fsm_loss += fsm_weights[layer]*torch.mean((fog_factor_b/(hb*wb) - fog_factor_a/(ha*wa))**2)/half/ b_feature.size(0)
399    loss_fsm += layer_fsm_loss / 4

The code is inconsistent with the paper, which one is correct?

**Comment by tiamjiakun** [comment]
If 'half' is placed in parentheses and squared, it seems that the code can remain consistent with the paper; Or is there a mistake in the paper?

**Comment by sohyun-l** [comment]
Thank you for your comment!
As you mentioned, 'half' should indeed be placed in parentheses and squared. However, since it is a constant, it will be incorporated into the hyperparameters of the loss function, and therefore, it will not impact the results.
